packages/bpk-component-pagination/src/BpkPaginationNudger.module.scss

```diff
@@ -23,7 +23,10 @@
   display: flex;
   width: tokens.bpk-spacing-xl() + tokens.bpk-spacing-sm();
   height: tokens.bpk-spacing-xl() + tokens.bpk-spacing-sm();
-  padding: tokens.$bpk-one-pixel-rem * 10;
+  padding-top: tokens.$bpk-one-pixel-rem * 2.75;
+  padding-right: tokens.$bpk-one-pixel-rem * 10;
+  padding-bottom: tokens.$bpk-one-pixel-rem * 10;
+  padding-left: tokens.$bpk-one-pixel-rem * 10;
```

Comment:
I think we can get rid of these, just writing up a demo

Reply:
Oops, have merged it - I'll create another PR once I can see what you are suggesting Mungo

Reply:
Yeah, no worries. I'll fire it across and see what you think.

Reply:
It also works with:
```
    padding-top: tokens.$bpk-one-pixel-rem * 2.75;
    justify-content: space-evenly;
```